commentHandler.go

```diff
@@ -308,3 +372,17 @@ func checkTransition(requestedAction string, currentState string) (string, bool)
 
 	return "", false
 }
+
+func nilMilestone(client *github.Client, ctx context.Context, URL string) (string, error) {
```

Comment:
possibly better to rename the function to `removeMilestone`.
please, remove the string return from `(string, error)` and leave only `error`.

there is no need to return a string in this case only an error (or nil) is OK.

Reply:
I agree, `removeMilestone` is better